src/Components/customEditor/editor.css

```diff
@@ -0,0 +1,3 @@
+#mainEditor {
```

Comment:
what is the purpose of this css `id`, remove it if you are not making use of it.